src/statsig/index.test.ts

```diff
@@ -103,6 +104,27 @@ describe('Statsig helpers', () => {
     })
   })
 
+  describe('getFeatureGate', () => {
+    it('returns default values if getting statsig feature gate throws error', () => {
+      ;(Statsig.checkGate as jest.Mock).mockImplementation(() => {
```

Comment:
guess this is copied from existing tests, but can we use the `mocked` util from `ts-jest/utils`?